bokeh/models/tiles.py

```diff
@@ -1,7 +1,6 @@
 from __future__ import absolute_import
 from ..plot_object import PlotObject
-
-from ..properties import (Any, Dict, Float, String, Int)
+from ..properties import (Any, Dict, Float, String, Int, Bool)
 
 class TileSource(PlotObject):
```

Comment:
On an unrelated basis, I'm curious if `TileSource` could have anything to do with an actual bokeh data source (`bokeh.models.sources.DataSource`).